src/ops/constant.rs

```diff
@@ -180,10 +212,7 @@ impl ConstValue {
 
     /// New 64 bit integer constant
     pub fn i64(value: i64) -> Self {
-        Self::Int {
-            value: value as HugrIntValueStore,
-            width: 64,
-        }
+        Self::Int(value as HugrIntValueStore)
```

Comment:
Is it worth keeping this? There's only thing about it that's `i64` is the argument type. You'll get a Rust type error rather than a constructor Error so I suppose that's slightly better but I'm not sure that's worth the implication of this method that the value is inherently 64-bit, e.g. if we made a Const out of a `ConstValue::i64` and `HashableType::Int(32)`....

Reply:
yeah good point, removed